app/scss/components/panel.scss

```diff
@@ -0,0 +1,23 @@
+.panel {
+    min-height: 300px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    box-sizing: border-box;
+    color: $lbh-white;
+    background: $lbh-dark-green;
```

Comment:
Better to import the Hackney colours from the pattern library. If you're importing the whole pattern library, you can find use `color: lbh-colours("lbh-white")` for example. The list of colour names available to use with `lbh-colours()` (and there hex codes) can be found here: https://github.com/LBHackney-IT/LBH-frontend/blob/master/src/lbh/settings/_colour-palette.scss

Reply:
OK done.